modules/strelka/1.0/strelka.smk

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env snakemake
+
+
+##### ATTRIBUTION #####
+
+
+# Original Author:  Lauren Chong
+# Module Author:    Helena Winata
+# Contributors:     N/A
+
+
+##### SETUP #####
+
+
+# Import package with useful functions for developing analysis modules
+import oncopipe as op
+
+# Setup module and store module-specific configuration in `CFG`
+# `CFG` is a shortcut to `config["lcr-modules"]["strelka"]`
+CFG = op.setup_module(
+    name = "strelka",
+    version = "1.0",
+    subdirectories = ["inputs", "strelka", "filtered", "outputs"]
+)
+
+# Define rules to be run locally when using a compute cluster
+localrules:
+    _strelka_input_bam,
+    _strelka_input_vcf,
+    _strelka_configure_paired,
+    _strelka_configure_unpaired,
+    _strelka_unzip,
+    _strelka_filter,
+    _strelka_output_vcf,
+    _strelka_all,
+
+
+##### RULES #####
+
+
+# Symlinks the input files into the module results directory (under '00-inputs/')
+rule _strelka_input_bam:
+    input:
+        bam = CFG["inputs"]["sample_bam"],
+        bai = CFG["inputs"]["sample_bai"]
+    output:
+        bam = CFG["dirs"]["inputs"] + "{seq_type}--{genome_build}/{sample_id}.bam",
+        bai = CFG["dirs"]["inputs"] + "{seq_type}--{genome_build}/{sample_id}.bam.bai"
+    run:
+        op.relative_symlink(input.bam, output.bam)
+        op.relative_symlink(input.bai, output.bai)
+
+
+rule _strelka_input_vcf:
+    input:
+        vcf = CFG["inputs"].get("candidate_indel")
+    output:
+        vcf = CFG["dirs"]["inputs"] + "{seq_type}--{genome_build}/vcf/{tumour_id}--{normal_id}--{pair_status}.candidateSmallIndels.vcf.gz"
+    conda:
+        CFG["conda_envs"]["tabix"]
+    shell:
+        "bgzip -c {input.vcf} > {output.vcf} && tabix -p vcf {output.vcf}"
```

Comment:
The output of Manta is already bgzip-compressed and tabix-indexed. I wonder if we can just point to this file before it gets decompressed by subsequent steps in the manta module (see example path below). I just don't know if Snakemake will properly configure the `_manta_run` rule as a dependency of this rule here. The [output of the `_manta_run` rule is a directory](https://github.com/LCR-BCCRC/lcr-modules/blob/master/modules/manta/2.0/manta.smk#L129), and this file would be within that directory, so I'm hoping that the dependency link would be created. Could you test that?

```
results/manta-1.0/02-manta/{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/results/variants/candidateSmallIndels.vcf.gz
```

Reply:
If this works, then we can just rely on symlinking here.

Reply:
So it doesn't create the dependency if we try to symlink the vcf.gz. It would still work if we call _manta_all and _strelka_all in the Snakefile.